Classes/Search/SearchBarSectionController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SearchBarSectionController.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 28/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import IGListKit
+
+protocol SearchBarSectionControllerDelegate: class {
+    func didFinishSearching(term: String?)
+    func didCancelSearching()
+}
+
+final class SearchBarSectionController: ListSectionController {
+
+    weak var delegate: SearchBarSectionControllerDelegate? = nil
```

Comment:
Tip you probably knew, but you do not actually need to set `= nil` for optional variables